plugins/generic/externalFeed/ExternalFeedPlugin.inc.php

```diff
@@ -1,375 +1,329 @@
-		}
-	}
-}
-
-?>
+<?php
+
+/**
+ * @file plugins/generic/externalFeed/ExternalFeedPlugin.inc.php
+ *
+ * Copyright (c) 2013-2018 Simon Fraser University Library
+ * Copyright (c) 2003-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class ExternalFeedPlugin
+ * @ingroup plugins_generic_externalFeed
+ *
+ * @brief ExternalFeedPlugin plugin class
+ */
+
+import('lib.pkp.classes.plugins.GenericPlugin');
+
+class ExternalFeedPlugin extends GenericPlugin {
+
+	const CUSTOM_STYLESHEET = 'externalFeedStyleSheet.css';
+
+	/**
+	 * Called as a plugin is registered to the registry
+	 * @param $category String Name of category plugin was registered to
+	 * @return boolean True iff plugin initialized successfully; if false,
+	 * 	the plugin will not be registered.
+	 */
+	public function register($category, $path) {
+		$success = parent::register($category, $path);
+
+		if (!Config::getVar('general', 'installed') || defined('RUNNING_UPGRADE')) return true;
+		if ($success && $this->getEnabled()) {
+			$this->import('classes.ExternalFeedDAO');
+			
+			HookRegistry::register('PluginRegistry::loadCategory', array($this, 'callbackLoadCategory'));
+			
+			$externalFeedDao = new ExternalFeedDAO($this->getName());
+			$returner =& DAORegistry::registerDAO('ExternalFeedDAO', $externalFeedDao);
+			
+			HookRegistry::register('Templates::Management::Settings::website', array($this, 'callbackShowWebsiteSettingsTabs'));
+			HookRegistry::register('LoadComponentHandler', array($this, 'setupGridHandler'));
+			
+			$templateMgr =& TemplateManager::getManager();
+			$templateMgr->addStyleSheet('externalFeed', '/' . $this->getStyleSheetFile());
+
+			// Journal home page display
+			HookRegistry::register('TemplateManager::display', array($this, 'displayHomepage'));
+
+			$this->_registerTemplateResource();
+		}
+		return $success;
+	}
+
+	/**
+	 * Get the display name of this plugin
+	 * 
+	 * @return string
+	 */
+	public function getDisplayName() {
+		return __('plugins.generic.externalFeed.displayName');
+	}
+
+	public function getDescription() {
+		return __('plugins.generic.externalFeed.description');
+	}
+
+	/**
+	 * Get the filename of the ADODB schema for this plugin.
+	 */
+	public function getInstallSchemaFile() {
+		return $this->getPluginPath() . '/' . 'schema.xml';
+	}
+
+	/**
+	 * Get plugin CSS URL
+	 *
+	 * @return string Public plugin CSS URL
+	 */
+	public function getCssUrl() {
+		return $this->getPluginPath() . '/css/';
+	}
+
+	/**
+	 * Get the filename of the default CSS stylesheet for this plugin.
+	 * @return string 
+	 */
+	public function getDefaultStyleSheetFile() {
+		return $this->getCssUrl() . 'externalFeed.css';
+	}
+
+	/**
+	 * Override the builtin to get the correct template path.
+	 * @return string
+	 */
+	public function getTemplatePath($inCore = false)
+	{
+		return $this->getTemplateResourceName() . ':templates/';
+	}
+
+	/**
+	 * @see Plugin::getActions()

```

Comment:
Better to use `@copydoc` instead of `@see`, unless you specify all the parameters/return type. `@see` just refers to the other function as a link; `@copydoc` brings in its documentation directly, incluidng parameters/return type.